src/components/Home.jsx

```
@@ -12,21 +11,47 @@ import { fetchFireStore } from "../Api";
 
 export default function Home() {
   const navigate = useNavigate();
+  const { setIsUser, setPlaylist } = useStore();
+
   const { setIsUser, setPlaylist, setLikedSongs } = useStore();
```

Biome (2.1.2)

[error] Shouldn't redeclare 'setIsUser'. Consider to delete it or rename it.

'setIsUser' is defined here:


(lint/suspicious/noRedeclare)